browser/ai_chat/pref_names.h

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_AI_CHAT_PREF_NAMES_H_
+#define BRAVE_BROWSER_AI_CHAT_PREF_NAMES_H_
+
+class PrefRegistrySimple;
+
+namespace ai_chat::prefs {
```

Comment:
why are the pref_names and constants in browser/ai_chat instead of components/ai_chat?

Reply:
moved now